docs/connectors/postgresql.asciidoc

```diff
@@ -2,5 +2,1044 @@
 :awestruct-layout: doc
 :linkattrs:
 :icons: font
+:source-highlighter: highlight.js
+:debezium-version: 0.4.0
 
-This connector is https://issues.jboss.org/browse/DBZ-3[incomplete]. Want to help us work on it? link:/docs/contribute[Learn how].
+Debezium's PostgreSQL Connector can monitor and record the row-level changes in the schemas of a PostgreSQL database. 
+
+The first time it connects to a PostgreSQL server/cluster, it reads a consistent snapshot of all of the schemas. When that snapshot is complete, the connector continuously streams the changes that were committed to PostgreSQL 9.6 or later and generates corresponding insert, update and delete events. All of the events for each table are recorded in a separate Kafka topic, where they can be easily consumed by applications and services.
+
```

Comment:
All the tests (including our Docker image) were done on 9.6

`9.4` was the first version where logical decoding was introduced, but it had several bugs which were fixed subsequently in 9.5 and 9.6

So, in theory we can say `9.4`, but IMO it's safer to start from `9.6`